tools/dreport.d/plugins.d/rbmc

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# config: 12345 22
+# @brief: Collect data-sync hosted DBus properties and current actively watched paths.
```

Comment:
Please avoid mentioning what we will collect, we will keep add more things in the future.